### ISSUE TITLE
3.2.1 Beta1 fails on MapleSim Parser

### ISSUE THREAD

**Issue by modelica-trac-importer** [modelica/Modelica#1130]
**Reported by dietmarw on 27 May 2013 12:27 UTC**
I just ran the MapleSim parser and it reports the following error: 
```
syntax error on line 376 of ./Media/Air/ReferenceMoistAir.mo: expected `=` but received `.`
```

----

Migrated-From: https://trac.modelica.org/Modelica/ticket/1130

**Comment by modelica-trac-importer** [comment]
**Comment by stefanv on 27 May 2013 14:25 UTC**
I believe the MapleSim parser is correct. The snippet of code giving the error is of the form:

```
v := F(function G(state.p), x, y, z);
```

The grammar for function arguments is:

```
function_argument :  
    "function" name "(" [ named_arguments ] ")"
    | expression 
```

Thus, the argument `function G(state.p)` is not valid. It should be `function G(something=state.p)`.

**Comment by modelica-trac-importer** [comment]
**Comment by otter on 28 May 2013 08:17 UTC**
Fixed in 725fad254d29276a22e2c6bcff1d50a8a7e5677f
